apps/site/src/app/(home)/sections/Sponsors/Sponsors.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @next/next/no-img-element */
+import Image from "next/image";
+
+import { getSponsors } from "./getSponsors";
+import { client } from "@/lib/sanity/client";
+import imageUrlBuilder from "@sanity/image-url";
+import fishingBoat from "@/assets/images/fishing-boat.png";
+
+import styles from "./Sponsors.module.scss";
+
+const builder = imageUrlBuilder(client);
+
+export default async function Sponsors() {
+	const sponsors = await getSponsors();
+
+	return (
+		<section className="container py-24 md:my-16 relative items-center flex flex-col md:p-8 w-3/4 mx-auto text-center">
+			<h2
+				className={`${styles.title} my-6 font-display sm:text-[3rem] text-[#fffce2] text-4xl text-center`}
+			>
+				Sponsors
+			</h2>
+			<p className="max-w-md mb-16 md:mb-12">
+				Interested in sponsoring IrvineHacks 2024? Check out our
+				information above to learn more about our event! For more
+				information, please email us at{" "}
+				<a href="mailto:hack@uci.edu">hack@uci.edu</a>
+			</p>
+			<div className="grid grid-cols-1 md:grid-cols-2 gap-8 items-center mb-40">
```

Comment:
Issue: The spacing between the text above and the sponsor images and the spacing between the sponsor images and the boat image on the bottom should be the same. Additionally, the sponsor images look a little close to each other, so it might be good to increase the gap. 

<img width="646" alt="image" src="https://github.com/HackAtUCI/irvinehacks-site-2024/assets/28026520/fbf61bdb-5bb3-4a8d-bf41-36b4236487df">